### PR TITLE
[SMALLFIX]Change Running-Alluxio-Fault-Tolerant-on-EC2.md

### PR DESCRIPTION
**Pull request by nju16szh**
Change Running-Alluxio-Fault-Tolerant-on-EC2.md

### CODE REVIEW

**Comment by nju16szh** [comment]
Jenkins, test this please.

File: docs/cn/deploy/Running-Alluxio-Fault-Tolerant-on-EC2.md [hunk 1]
```diff
@@ -35,7 +35,10 @@ vagrant box add dummy https://github.com/mitchellh/vagrant-aws/raw/master/dummy.
 
 进入`deploy/vagrant`目录下，运行：
 
-{% include Running-Alluxio-Fault-Tolerant-on-EC2/install-vagrant.md %}
+```bash
+sudo bash bin/install.sh
+```
+
```
Comment (aaudiber) [inline docs/cn/deploy/Running-Alluxio-Fault-Tolerant-on-EC2.md]: please remove the extra newline

File: docs/cn/deploy/Running-Alluxio-Fault-Tolerant-on-EC2.md [hunk 2]
```diff
@@ -51,7 +54,11 @@ sudo pip install -r pip-req.txt
 接着创建[access keys](https://aws.amazon.com/developers/access-keys/)并且设置`AWS_ACCESS_KEY_ID`和
 `AWS_SECRET_ACCESS_KEY`环境变量:
 
-{% include Running-Alluxio-Fault-Tolerant-on-EC2/access-keys.md %}
+```bash
+export AWS_ACCESS_KEY_ID=<your access key>
+export AWS_SECRET_ACCESS_KEY=<your secret access key>
+```
+
```
Comment (aaudiber) [inline docs/cn/deploy/Running-Alluxio-Fault-Tolerant-on-EC2.md]: same here

**Comment by nju16szh** [comment]
Jenkins, test this please.

**Comment by aaudiber** [comment]
alluxio-bot, merge this please

**Pull request merged**